src/components/can_cooperation/reverse_sdl_pt.json

```diff
@@ -1,6 +1,66 @@
 {
     "policy_table": {
         "module_config": {
+            "equipment": {
+              "zones": {
+                "Driver": {
+                  "col": 0,
+                  "row": 0,
+                  "level": 0,
+                  "auto_allow": {
+                    "RADIO": {},
+                    "CLIMATE": {}
+                  },
+                  "driver_allow": {}
+                },
+                "Front Passenger": {
+                  "col": 1,
+                  "row": 0,
+                  "level": 0,
+                  "auto_allow": {
+                    "CLIMATE": {}
+                  },
+                  "driver_allow": {
+                    "RADIO": {}
+                  }
+                },
+                "Left Rare Passenger": {
+                  "col": 0,
+                  "row": 1,
+                  "level": 0,
+                  "auto_allow": {
+                    "RADIO": {
+                      "ButtonPress": [],
+                      "GetInteriorVehicleDataCapabilities": [],
+                      "GetInteriorVehicleData": ["moduleDescription"]
+                    }
+                  },
+                  "driver_allow": {
+                    "RADIO": {
+                      "ButtonPress": ["ButtonName", "ButtonPressMode"],
```

Comment:
I didn't understand your question. But this TRS hasn't been analyzed full yet. So it will be changed later I think.